web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -118,13 +119,12 @@ export default function ZoneDetails(): JSX.Element {
               displayByEmissions={displayByEmissions}
             />
           )}
+
           <MethodologyCard />
+          <HorizontalDivider />
+          <div className="mb-1 text-sm font-semibold">Looking for forecast data?</div>
```

Comment:
"Looking for forecast data" and the CTA feel a little disconnected. Maybe they could go on the same line? 🤔 
<img width="368" alt="Screenshot 2024-12-19 at 10 53 22" src="https://github.com/user-attachments/assets/7689ccba-0c62-4f43-8857-c8f653cd3763" />

Reply:
I agree, this was quite a speculative change, we can remove it or try on the same line